src/libs/OptionsListUtils.ts

```diff
@@ -1438,8 +1474,7 @@ function getOptions(
     let recentReportOptions = [];
     let personalDetailsOptions: ReportUtils.OptionData[] = [];
     const reportMapForAccountIDs: Record<number, Report> = {};
-    const parsedPhoneNumber = PhoneNumber.parsePhoneNumber(LoginUtils.appendCountryCode(Str.removeSMSDomain(searchInputValue)));
-    const searchValue = parsedPhoneNumber.possible ? parsedPhoneNumber.number?.e164 : searchInputValue.toLowerCase();
+    const {output: searchValue, parsedPhoneNumber} = getSearchValueForPhoneOrEmail(searchInputValue);
```

Comment:
I don't think adding parsedPhoneNumber to the result of `getSearchValueForPhoneOrEmail` function is good. Because we only use `parsedPhoneNumber` in here. Let's keep `getSearchValueForPhoneOrEmail` function as before and use it here
```
const parsedPhoneNumber = PhoneNumber.parsePhoneNumber(LoginUtils.appendCountryCode(Str.removeSMSDomain(searchInputValue)));
const {output: searchValue, parsedPhoneNumber} = getSearchValueForPhoneOrEmail(searchInputValue);
```

Reply:
I just decided to revert this change, because parsePhoneNumber might have negative impact on the performance (it's doing operations based on regexes under the hood). Once all the lists migrate to this type of search, all the code related to search from this function could be possibly removed, including this one, so I think we could leave it unchanged for now and remove in the future, to prevent excessive function calls